workspace/packages/local/Deck/classic/src/view/main/Main.js

```diff
@@ -41,25 +42,4 @@ Ext.define('Deck.view.main.Main', {
         },
         region: 'center'
     }]
-});
-
-
-Ext.define('MyApp.view.', {
-    extend: 'Ext.app.ViewModel',
-    alias: 'viewmodel.myalias',
-
-    data: {},
-    formulas: {},
-    stores: {}
-
-
-});
-
-Ext.define('MyApp.view.', {
-    extend: 'Ext.app.ViewController',
-    alias: 'controller.myalias',
-
-    init: function() {},
-    initViewModel: function(vm) {},
-
-});
```

Comment:
These two classes where causing trouble with the build, they have the same name (empty string).